.github/workflows/botonic-publish.yml

```diff
@@ -0,0 +1,372 @@
+name: Botonic Publish
+
+on:
+  workflow_dispatch:
+    inputs:
+      version_type:
+        description: 'Type of version upgrade'
+        required: true
+        default: 'patch'
+        type: choice
+        options:
+          - 'patch'
+          - 'minor'
+          - 'major'
```

Comment:
I would remove 'major' (or leave it commented) at this moment as we are only publishing "minor" and patch versions so we don't mess publishing a 1.X.Y